spec/models/payment/go_cardless/subscription_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe Payment::GoCardless::Subscription do
+
+  let(:subscription) { build :payment_go_cardless_subscription }
+  subject { subscription }
+
+  it { is_expected.to respond_to :go_cardless_id }
+  it { is_expected.to respond_to :amount }
+  it { is_expected.to respond_to :currency }
+  it { is_expected.to respond_to :status }
+  it { is_expected.to respond_to :name }
+  it { is_expected.to respond_to :created_at }
+  it { is_expected.to respond_to :updated_at }
+
+  # Associations
+  it { is_expected.to respond_to :page_id }
+  it { is_expected.to respond_to :page }
+  it { is_expected.to respond_to :action }
+  it { is_expected.to respond_to :action_id }
+  it { is_expected.to respond_to :payment_method }
+  it { is_expected.to respond_to :payment_method_id }
+  it { is_expected.to respond_to :customer }
+  it { is_expected.to respond_to :customer_id }
+
+  # Fields passed back from GoCardless that we don't record
+  it { is_expected.not_to respond_to :start_date }
+  it { is_expected.not_to respond_to :end_date }
+  it { is_expected.not_to respond_to :interval }
+  it { is_expected.not_to respond_to :interval_unit }
+  it { is_expected.not_to respond_to :day_of_month }
+  it { is_expected.not_to respond_to :month }
```

Comment:
See my comments and questions on your original GoCardless issue in GitHub. What are the options for the transactional emails that we are required to send at least some DD customers? If we are doing these ourselves, it seems to me we definitely need the above information. May well be missing something, but payment interval information seems potentially important so we know the date their next charge is happening (and crucial if we are handling the transactional emails ourselves).